pep-0702.rst

```diff
@@ -170,6 +172,17 @@ compatible with future versions of Python) would want to catch any use of
 deprecated functionality in their CI pipeline. Therefore, it is recommended
 that type checkers provide configuration options that cover both use cases.
 
+Deprecation policy
+------------------
+
+CPython's deprecation policy (:pep:`387`) will require that new deprecations
```

Comment:
This is what's being _proposed_, right?

Reply:
Yes, should that be made more obvious somehow?

Reply:
I had to read it twice before I was sure of it. Maybe:
```suggestion
We propose that CPython's deprecation policy (:pep:`387`) is updated to require that new deprecations
```